model/nc-news-models.js

```diff
@@ -26,17 +26,24 @@ function fetchArticle(article_id) {
 }
 
 function fetchAllArticles() {
-    const articleQuery = db.query(`SELECT * FROM articles`)
+    const articleQuery = db.query(`SELECT * FROM articles ORDER BY created_at DESC`)
```

Comment:
is there a way we can do this with only one query? Remember we're working with relational databases and we have the JOIN keyword to take advantage of!